hooks/useGovernanceAssets.ts

```diff
@@ -162,6 +164,41 @@ export default function useGovernanceAssets() {
         canUseTokenTransferInstruction &&
         realm?.account.config.useCommunityVoterWeightAddin,
     },
+    {
+      id: Instructions.CreateAssociatedTokenAccount,
+      name: 'Create Associated Token Account',
+      isVisible: canUseAnyInstruction,
+    },
```

Comment:
is there a specific reason you need an instruction to create ATA's? we usually just create them in preflight instructions

Reply:
I think we had the case where we wanted the ATA creation to be visible in the proposal, but can't recall for sure. Would you rather see that instruction removed?